cmd/kube-proxy/app/server.go

```diff
@@ -163,33 +164,33 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(cliflag.NewMapStringBool(&o.config.FeatureGates), "feature-gates", "A set of key=value pairs that describe feature gates for alpha/experimental features. "+
```

Comment:
This one can be replaced with [this](https://github.com/kubernetes-sigs/legacyflag/blob/master/pkg/legacyflag/map_string_bool.go)

Reply:
Thank you for your comment! Let me dive a bit more into the cliflag pkg to get a better understanding and i''ll get this fixed.

Reply:
Let's use the legacyflag MapStringBoolVar for this, since it has built-in support for merge functionality: https://github.com/kubernetes-sigs/legacyflag/blob/master/pkg/legacyflag/map_string_bool.go